src/Files/MultilingualResources/Files.ar.xlf

```diff
@@ -1576,31 +1576,31 @@
         </trans-unit>
         <trans-unit id="SidebarCloudDrives" translate="yes" xml:space="preserve">
           <source>Cloud Drives</source>
-          <target state="new">Cloud Drives</target>
+          <target state="needs-review-translation">محركات السحابة</target>
```

Comment:
If the translations are good, can you change the state to `translated`?

Reply:
I thought it was inappropriate but after thinking I think it is, at least for the time being, so I will change the state now, and I may reconsider some of the other strings

Reply:
I fixed the issue, and fixed some other bugs, thanks <3